nucleus/diagnostics/context/src/test/java/org/glassfish/diagnostics/context/impl/ContextManagerImplUnitTest.java

```diff
@@ -77,7 +77,8 @@ public void registerContextFactoryForPrefixNamed(String prefixName, ContextViewF
 			}
 		};
 
-		ContextManagerImpl cmi = new ContextManagerImpl();
+		@SuppressWarnings("unused")
+        ContextManagerImpl cmi = new ContextManagerImpl();
```

Comment:
Indentation

Reply:
It's indented allright (seemingly, on my screen), but it looks like the original code used tabs. I'll change them all to spaces